TypeCobol/Compiler/SqlCodeElements/SqlStatementElement.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections;
+using System.Collections.Generic;
+using TypeCobol.Compiler.CodeElements;
+
+namespace TypeCobol.Compiler.SqlCodeElements
+{
+    /// <summary>
+    /// Base class for all Sql statement CodeElements
+    /// </summary>
+    public abstract class SqlStatementElement : StatementElement
+    {
+        protected SqlStatementElement(SqlCodeElementType codeElementType, SqlStatementType statementType) 
+            : base((CodeElementType)codeElementType, (StatementType)statementType)
```

Comment:
It works but only for CommitStatement because only one value is ahrde between enums. So if we introduce other statements/constructs, the enum casts won't work. I think we have no choice but to extend existing enums.